tests/Rector/Class_/CommandPropertyToAttributeRector/Fixture/commandWithHidden.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\Class_\CommandPropertyToAttributeRector\Fixture;
+
+class CommandWithHidden extends \Symfony\Component\Console\Command\Command
+{
+    /** @var string|null */
+    public static $defaultName = 'sunshine';
+
+    public function configure(): void
+    {
+        $this
+            ->setHidden();
```

Comment:
setHidden(true) test case need to be added, are they need to be removed whatever value pass on it?

Reply:
Added testcase with `setHidden(true)` false case is handled in https://github.com/rectorphp/rector-symfony/pull/259/files#diff-1ce73e8d15ebe88b93e9a7d208697815fcadb4ab67e8067a0e646e9bee4141db

What do you mean with `are they need to be removed whatever value pass on it?`?

Reply:
There are 3 possible conditions:

- setHidden()
- setHidden(false)
- setHidden(true)

Are they need to be removed so need check against Arg value ?

Reply:
I think what you mean is do we need to remove `setHidden` function on all these cases and move them to the `AsCommand` attribute? I would say yes. But not sure that is what you mean

Reply:
Yes, that's what I mean 👍 , it seems test added now, looks good 👍